mhctools/base_commandline_predictor.py

```diff
@@ -55,11 +55,16 @@ def __init__(
                 raise SystemError("Failed to run %s" % command)
             valid_alleles = None
 
-        BasePredictor.__init__(
-            self,
-            alleles,
-            epitope_lengths,
-            valid_alleles=valid_alleles)
+        try:
+            BasePredictor.__init__(
+                self,
+                alleles,
+                epitope_lengths,
+                valid_alleles=valid_alleles)
+        except UnsupportedAllele as e:
+            additional_message = ("\nRun command %s %s to see a list of valid alleles" % (
+                command, supported_allele_flag) if supported_allele_flag else "")
+            raise UnsupportedAllele(e.message + additional_message)
```

Comment:
Looks good except that I'm not sure that Py3 has a `message` field on ValueErrors